assessments/2022-12-02-booleans/code.js

```diff
@@ -0,0 +1,24 @@
+const needHeavyCoat = (outside, cold) => {
+  return outside && cold;
+}
+const needSunscreen = (beach, ski) => {
+  return beach && ski;
+}
+const needMittens = (outside, warm) => {
+  return outside && !warm;
+}
+const isVenomous = (striped, blueHead) => {
+  return striped || !striped || !blueHead;
```

Comment:
Quite close. However anything of the form `x || !x` is always true because if `x` is `true` the expression as a whole is true because of the first part and if `x` is false, it's true because of the second part. 


A correct expressions would be something like:

```javascript
  return striped || !blueHead;
```

or possibly you were going for something like:

```suggestion
  return striped || (!striped && !blueHead);
```

I.e. it's striped (all striped snakes are venomous) or it's not stripped and it's not blue headed (of the not-striped snakes, only the blue-headed Blobby snake is not venomous.)

However the first one gets to the same thing because the only way the expression would end up depending on what's after `striped` is if `striped` is `false` so you don't need to check `!striped`.